src/BenchmarksApps/TodosApi/TodosApi.csproj

```diff
@@ -9,12 +9,27 @@
     <LangVersion>preview</LangVersion>
     <UserSecretsId>b8ffb8d3-b768-460b-ac1f-ef267c954c85</UserSecretsId>
     <PublishAot>true</PublishAot>
-    <EnableLogging Condition="$(Configuration.StartsWith('Debug'))">true</EnableLogging>
+    <EnableConfigurationBindingGenerator>false</EnableConfigurationBindingGenerator>
+    <EnableLogging Condition=" '$(EnableLogging)' == '' and $(Configuration.StartsWith('Debug')) ">true</EnableLogging>
+    <EnableOpenApi Condition=" '$(EnableOpenApi)' == '' and $(Configuration.StartsWith('Debug')) ">true</EnableOpenApi>
     <DefineConstants Condition=" '$(EnableLogging)' == 'true' ">$(DefineConstants);ENABLE_LOGGING</DefineConstants>
+    <DefineConstants Condition=" '$(EnableOpenApi)' == 'true' ">$(DefineConstants);ENABLE_OPENAPI</DefineConstants>
   </PropertyGroup>
-
+  
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.0-preview.*" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.0-preview.*" />
     <PackageReference Include="Npgsql" Version="8.0.0-preview.2" />
+    <Content Update="appSettings.Development.json" CopyToPublishDirectory="false" />
+  </ItemGroup>
+
+  <PropertyGroup Condition=" '$(EnableOpenApi)' == 'true' ">
```

Comment:
I'm a bit torn on this. We explicitly cut Open API from our .NET 8 target. So I wonder if this is just clutter.

Reply:
Yeah I'm exploring if there's a flow here where OpenAPI is enabled just so that we can do build-time document generation, but not in the app when published (and thus not impacted by native AOT). It very likely needs some more iteration and arguably could be done in a different place other than this app, but this is the one that's tracked right now.